components/elm/src/biogeophys/CanopyHydrologyMod.F90

```diff
@@ -382,7 +385,7 @@ subroutine CanopyHydrology(bounds, &
           ! Note that it's still possible that (some of) this irrigation water will runoff (as runoff is computed later)
           if(irrigate) then
            if (tw_irr) then ! else one way  
-               qflx_supply(p) = atm2lnd_vars%supply_grc(g)*(1-wt_lunit(g,istdlak))/pgwgt(p) ! original supply at grid level (mm/s) concentrate to pft level. Take lake fraction into consideration
+               qflx_supply(p) = atm2lnd_vars%supply_grc(g)*(1-wt_lunit(g,tpu_ind,istdlak))/pgwgt(p) ! TKT changed (1-wt_lunit(g,istdlak))/pgwgt(p) ! original supply at grid level (mm/s) concentrate to pft level. Take lake fraction into consideration
```

Comment:
This !TKT comment not needed - already captured by commit

Reply:
@thorntonpe I have removed comment in my latest commit. Thanks.